Tests/swift-sdk-swift-tests/IterableAPITests.swift

```diff
@@ -589,24 +590,20 @@ class IterableAPITests: XCTestCase {
         let config = IterableConfig()
         IterableAPI.initializeForTesting(apiKey: IterableAPITests.apiKey, config: config, networkSession: networkSession)
         IterableAPI.email = "user@example.com"
-        IterableAPI.get(
```

Comment:
Why only this method? :) Looks like formatting is inconsistent in the other methods too.